Bottom: 6ab745679ece8b90071f79069f035b5bdd963379
Top:    d48abb1d1214e1255c982084181d0ecc75d57fa0
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-09 14:28:21 -0700

Refresh of python-fix

---
diff --git a/lldb/CMakeLists.txt b/lldb/CMakeLists.txt
index 317643a4422..7ddb552968c 100644
--- a/lldb/CMakeLists.txt
+++ b/lldb/CMakeLists.txt
@@ -39,7 +39,7 @@ endif()
 if (NOT LLDB_DISABLE_PYTHON)
   execute_process(
     COMMAND ${PYTHON_EXECUTABLE}
-        -c "import distutils.sysconfig; print(distutils.sysconfig.get_python_lib(False, False, ''))"
+        -c "import distutils.sysconfig; print(distutils.sysconfig.get_python_lib(False, False, 'foo')[4:])"
     OUTPUT_VARIABLE LLDB_PYTHON_DEFAULT_RELATIVE_PATH
     OUTPUT_STRIP_TRAILING_WHITESPACE)
